src/UserLogin.ts

```diff
@@ -0,0 +1,25 @@
+import * as consts from "./Consts"
+import getClient from "./Client"
+
+export async function getUser(): Promise<string> {
+  const userIdFromLocalStorage: string | null = localStorage.getItem(
+    consts.brigadeUserIdKey
+  )
+
+  if (userIdFromLocalStorage) {
+    return userIdFromLocalStorage
+  }
+
+  try {
+    const fetchedUserId = (await getClient().authn().whoAmI()).id
+    localStorage.setItem(consts.brigadeUserIdKey, fetchedUserId)
+
+    return fetchedUserId
+  } catch {
+    return ""
+  }
+}
+
+export function removeUser(): void {
+  localStorage.removeItem(consts.brigadeUserIdKey)
+}
```

Comment:
I'm a bit iffy on this...

I really don't want anyone/anything except the logout callback clearing this info out of local storage, so exporting it might not be the best thing to do.

Can we just replace the `removeUser()` call in the logout callback with `ocalStorage.removeItem(consts.brigadeUserIdKey)`?